scripts/pagerduty.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+# Pagerduty constants
+LOGFILE="/home/tmp_log/log-20190628.153354/*.log"
+LOG=$(tail -n 1000 $LOGFILE)
+SHARD_NUM=$(echo "$LOG" | grep -oE -m 1 "Shard\":.?" | rev | cut -c 1)
+IP=$(dig -4 @resolver1.opendns.com ANY myip.opendns.com +short)
+KEY="5d11f6173899423689b6fc98691930de"
+TRIGGER="\
+{\
+  \"routing_key\": \"$KEY\",\
+  \"event_action\": \"trigger\",\
+  \"dedup_key\": \"$IP\",\
+  \"payload\": {\
+    \"summary\": \"Node NOT validating: Shard-$SHARD_NUM: $IP\",\
+    \"source\": \"$IP\",\
+    \"severity\": \"critical\"\
+  }\
+}"
+RESOLVE="\
+{\
+  \"routing_key\": \"$KEY\",\
+  \"event_action\": \"resolve\",\
+  \"dedup_key\": \"$IP\"\
+}"
+JSON_HEADER="Content-Type: application/json"
+POSTURL="https://events.pagerduty.com/v2/enqueue"
+
+# Validation check constants
```

Comment:
Can you make this script not hang even if the incident has been manually resolved?

Reply:
I found out it doesn't actually stall, seems like something was wrong with the code before and I fixed it in one of the prior commits.